### ISSUE TITLE
Deprecate non-deflate serialization

### ISSUE THREAD

**Issue by jonhoo** [HdrHistogram/HdrHistogram_rust#90]
As far as I can tell, the V2 log format in the Java implementation does not support _not_ compressing the histograms. Should we remove support for that? Should we at least update the examples to prefer the more "standard" serializer that will interoperate with other hdrhistogram libraries?

cc @marshallpierce

**Comment by marshallpierce** [comment]
Yep, at the very least updating examples seems reasonable. Maybe people use uncompressed ones in practice? 🤔 Weird, but possible...

**Comment by jonhoo** [comment]
I discovered today that the Python bindings don't even support uncompressed histograms. So was surprised by that for a bit when my histograms wouldn't parse.
